src/js/themes/base.js

```diff
@@ -1436,6 +1436,61 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         // extend: undefined,
       },
     },
+    tag: {
+      size: {
+        /*
+        small: {
+          border: {
+            radius: `${baseSpacing * 0.75}px`,
+          },
+          pad: {
+            vertical: `${baseSpacing / 4 - borderWidth}px`, // 4px
+            horizontal: `${baseSpacing - borderWidth * 2}px`, // 20px,
+          },
+        },
+        medium: {
+          border: {
+            radius: `${baseSpacing * 0.75}px`, // 18px
+          },
+          pad: {
+            vertical: `${baseSpacing / 4 - borderWidth}px`, // 4px
+            horizontal: `${baseSpacing - borderWidth}px`, // 22px
+          },
+        },
+        large: {
+          border: {
+            radius: `${baseSpacing}px`, // 24px
+          },
+          pad: {
+            vertical: `${baseSpacing / 4 + borderWidth}px`, // 8px
+            horizontal: `${baseSpacing + borderWidth * 4}px`, // 32px,
+          },
+        }, 
+        */
+      },
+      container: {
+        align: 'center',
+        border: true,
+        direction: 'row',
+        round: 'large',
+        pad: {
+          horizontal: `${baseSpacing / 4 - borderWidth}px`,
+        },
+      },
+      gap: 'xsmall',
+      name: { },
+      padding: {
```

Comment:
What drives putting something inside `tag.container` vs. just at `tag`?

Reply:
> What drives putting something inside `tag.container` vs. just at `tag`?

`tag.container` is applied to the very outside component which is either a Box or a Button depending on if its onClick-able or not. It has the border styling/shape and some basic padding.

The rest of the stuff in tag ends up on the `name:value` content and remove button.

Any thoughts on how to structure this?

Reply:
How is `theme.tag.container.pad` different from `theme.tag.padding`? I can't tell by looking at the theme. I'm wondering if we can simplify it to:
```
tag: {
  name: { },
  value: { },
  gap:
  separator:
  pad:
  round:
  background:
  border:
},
```
or whether we should go towards:
```
tag: {
  name: { },
  value: { },
  container: {
    gap:
    separator:
    pad:
    round:
    background:
    border:
  },
},
```